Libraries/src/Amazon.Lambda.RuntimeSupport/Client/InternalClientAdapted.cs

```diff
@@ -243,11 +243,11 @@ public async System.Threading.Tasks.Task<SwaggerResponse<System.IO.Stream>> Next
                     try
                     {
                         var headers_ = System.Linq.Enumerable.ToDictionary(response_.Headers, h_ => h_.Key, h_ => h_.Value);
-                        if (response_.Content != null && response_.Content.Headers != null)
-                        {
-                            foreach (var item_ in response_.Content.Headers)
+
+                        // NOTE: response_.Content can never be null (See: https://github.com/dotnet/runtime/blob/main/src/libraries/System.Net.Http/src/System/Net/Http/HttpResponseMessage.cs#L42)
```

Comment:
I'm good with the change but we don't need the comments. Those links will get out of date overtime so I would rather not have them.

Reply:
Done